src/R/Support/Test/Functions/FunctionInfoTest.cs

```diff
@@ -62,13 +64,25 @@ public class FunctionInfoTest : IAsyncLifetime {
             functionInfo.Should().NotBeNull();
             functionInfo.Name.Should().Be("eval");
             functionInfo.Description.Should().NotBeEmpty();
-            functionInfo.Signatures.Should().ContainSingle()
-                .Which.Arguments.Should().HaveCount(4);
+            functionInfo.Signatures.Should().ContainSingle().Which.Arguments.Should().HaveCount(4);
 
             List<int> locusPoints = new List<int>();
-            string signature = functionInfo.Signatures[0].GetSignatureString(locusPoints);
+            string signature = functionInfo.Signatures[0].GetSignatureString("eval", locusPoints);
             signature.Should().Be("eval(expr, envir = parent.frame(), enclos = if(is.list(envir) || is.pairlist(envir)) parent.frame() else baseenv(), n)");
             locusPoints.Should().Equal(5, 11, 35, 116, 117);
         }
+
+        [Test]
+        public async Task AliasesTest() {
+            var functionInfo = await PackageIndexUtility.GetFunctionInfoAsync(_functionIndex, "select");
+
+            functionInfo.Should().NotBeNull();
+            functionInfo.Name.Should().Be("lm.ridge");
```

Comment:
We need to check with the user who reported it that this is what they actually expect. I suspect that a lot more people use `dplyr` than the package where this comes from - we might have to special-case that for now (and come up with some long term plan to fix it in general).